src/entity/posts/Post.ts

```diff
@@ -215,10 +211,6 @@ export class Post {
   downvotes: number;
 
   @Column({ type: 'jsonb', default: {} })
-  @Index('IDX_post_flags_sentAnalyticsReport', { synchronize: false })
-  @Index('IDX_post_flags_banned', { synchronize: false })
-  @Index('IDX_post_flags_deleted', { synchronize: false })
-  @Index('IDX_post_flags_promoteToPublic', { synchronize: false })
```

Comment:
I think these should not be deleted, its just a matter of we still not using the flags instead of columns.

Reply:
At this point they don't ever hit according to the query though, so either they wrong or not used.

Reply:
They are not used. We have to make not to return them when we start using `flags.deleted` instead of `deleted` column (whenever that happens).

Reply:
Ah gotcha :)

Reply:
Let's reintroduce separately then once we do

Reply:
if we know we gonna use them in the near future, can we keep them please? Introducing indexes on the posts table can be expensive

Reply:
We can, but think they'll be different anyway.

Lets leave them existing for now 👍

Reply:
Your call then. If you want remove them

Reply:
> Introducing indexes on the posts table can be expensive

with `CONCURRENCLY` shouldn't be.